index.bs

```diff
@@ -3666,6 +3684,21 @@ credential.
         When storing a {{PublicKeyCredentialUserEntity/displayName}} member's value,
         the value MAY be truncated as described in [[#sctn-strings-truncation]]
         using a size limit greater than or equal to 64 bytes.
+
+    :   <dfn>language</dfn>
+    ::  An OPTIONAL [=resource-wide default=] language field [[!String-Meta]] for the entity.
```

Comment:
Does the word "entity" clearly mean the entire `PublicKeyCredentialUserEntity`? Or should this say something like "... for all natural language strings contained by the entity" to clarify?

Reply:
Yes, that was the intent. Does 1bcd085855738204987ee00dae5430e67c701a85 make this clearer?1